### DISCUSSION TITLE
How to set log level for IoTDB cluster?

### DISCUSSION THREAD

**Discussion by chengjianyun** [apache/iotdb#3785] [Q&A] [answered]
Hi, there

I'd like to change the log level of some packages for IoTDB cluster, I made some change in conf/logback.xml. But it doesn't take effect. Could anyone help?

Thanks!

**Comment by ijihang** [comment, marked as answer]
Have you tried to change < root level = "info" > in logback.xml to < root level = "debug" >

**Reply by chengjianyun** [reply]
Yes, I did do that but didn't take effect. Does it work on your side?

**Reply by ijihang** [reply]
Yes, it takes effect, and the debug log will be output to the debug.log file

**Reply by chengjianyun** [reply]
Thanks, @ijihang. I think it needs to update the `logback.xml` while `mvn package`. Just update the `logback.xml` under `config` folder in the running environment doesn't take effect.
